### PR TITLE
Update SSWG members

### PR DESCRIPTION
**Pull request by jdmcd**

### CODE REVIEW

**Review by FranzBusch — APPROVED** [review]

**Comment by adam-fowler** [comment]
Dave Moser's not on that list either.

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The current Swift Server work group consists of the following people:
 * Patrick Freed ([@patrickfreed](https://github.com/patrickfreed), MongoDB)
 * Simon Pilkington ([@tachyonics](https://github.com/tachyonics), Amazon)
 * Tim Condon ([@0xTim](https://github.com/0xTim), Vapor)
-* Todd Varland ([@toddvarland](https://github.com/toddvarland), Amazon)
+* Jimmy McDermott ([@jdmcd](https://github/com/jdmcd), Transeo)
```
Comment (tomerd) [inline README.md:21]: should this point to swift.org version of this file? I think there is some duplicate info and would be better to have one source of truth

Comment (tomerd) [reply README.md:21]: https://www.swift.org/sswg/

Comment (jdmcd) [reply README.md:21]: Yeah I think that would be smart - opened a PR: https://github.com/swift-server/sswg/pull/79

**Pull request merged**